src/libNode/Node.cpp

```diff
@@ -52,14 +52,13 @@ void addBalanceToGenesisAccount()
 {
     LOG_MARKER();
 
-    const uint256_t bal{100000000000};
+    const uint256_t bal{10000};
```

Comment:
Any specific reason to do this ?

Reply:
Will put the number back. Did this change for simplification on local unit test.